### PR TITLE
fix: Remove all references to v2

### LINKED ISSUE #4553: Remove all refences to v2
# 🚀 Tech Debt

### Relevant Package [**REQUIRED**]

 This feature request is for al code, comments and readme

### Description [**REQUIRED**]
"FYI there are a "fair number" of "v2" strings hanging around"

### Describe the solution you'd like
Remove all references to v2. Do not change them to v3 unless absolutely needed.

### Describe alternatives you've considered
 Have you considered any alternative solutions or workarounds?

### PR DESCRIPTION
**Pull request by FelixTing**
fix #4553 

**If your build fails**  due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/edgex-go/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] I am not introducing a breaking change (if you are, flag in conventional commit message with `BREAKING CHANGE:` describing the break)
- [x] I am not introducing a new dependency (add notes below if you are)
- [ ] I have added unit tests for the new feature or bug fix (if not, why?)
- [ ] I have fully tested (add details below) this the new feature or bug fix (if not, why?)
- [ ] I have opened a PR for the related docs change (if not, why?)
  <link to docs PR>

### CODE REVIEW

**Review by cloudxxx8 — DISMISSED** [review]
LGTM

File: internal/security/spiffetokenprovider/init.go [hunk 1]
```diff
@@ -164,7 +164,7 @@ func (b *Bootstrap) BootstrapHandler(ctx context.Context, _ *sync.WaitGroup, _ s
 	//
 
 	// Handle gettoken endpoint
-	http.HandleFunc("/api/v2/gettoken", func(w http.ResponseWriter, r *http.Request) {
+	http.HandleFunc("/api/v3/gettoken", func(w http.ResponseWriter, r *http.Request) {
```
Comment (lenny-intel) [inline internal/security/spiffetokenprovider/init.go:167]: This should be using the API  constants from core contracts
```suggestion
	http.HandleFunc(common.ApiBase +"/gettoken", func(w http.ResponseWriter, r *http.Request) {
```

**Review by lenny-intel — CHANGES_REQUESTED** [review]
These fields should be removed.
https://github.com/edgexfoundry/edgex-go/blob/main/internal/support/notifications/config/config.go#L44-L45

File: internal/security/spiffetokenprovider/init.go (hunk 1 above)
Comment (FelixTing) [reply internal/security/spiffetokenprovider/init.go]: updated

**Comment by FelixTing** [comment]
> These fields should be removed. https://github.com/edgexfoundry/edgex-go/blob/main/internal/support/notifications/config/config.go#L44-L45

removed

**Review by lenny-intel — APPROVED** [review]
LGTM

**Pull request merged**